### PR TITLE
Tinashechiraya patch 7

### PR DESCRIPTION
**Pull request by tinashechiraya**
[Screencast from 09-09-2023 09:33:04.webm](https://github.com/kartoza/sawps/assets/70011086/f8ca3409-facf-4f19-92ac-5cce970b4925)

Description:
I have placed the remember me checkbox next to the remember text 
I have fixed ,error editing reminder issue
I have placed the reveal tokens in a try catch so it doesn't break the code when certain variables it requires are undefined
I have stretched the width of the buttons on the organisations page so they don't wrap the text

### CODE REVIEW

File: django_project/stakeholder/test_view.py [hunk 1]
```diff
@@ -389,6 +395,27 @@ def test_add_reminder_and_schedule_task_success(self):
...
+        self.factory = RequestFactory()
+        request = self.factory.post(url, data)
+        request.user = self.user
+
+        view = RemindersView()
+
+        response = view.add_reminder_and_schedule_task(request)
+
+        # Check the response status code and content
+        self.assertEqual(response.status_code, 200)
```
Comment (dimasciput) [inline django_project/stakeholder/test_view.py:417]: Please always test the data too

**Review by dimasciput — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 28 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 27 min

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 29 min

**Pull request merged**